### ISSUE TITLE
SA1008 not reported inside cref attributes

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1585]
#### Sample code:

```csharp
/// <see cref="Method ()"/>
void Method()
{
}
```

#### Expected result:

SA1008 is reported, and the code fix removes the space between `Method` and `(`.

#### Actual result:

No diagnostic is reported; no code fix is available.

**Comment by Noryoko** [comment]
Grabbing this.
